yarn.lock

```diff
@@ -10526,7 +10526,7 @@ core-js@^2.4.0, core-js@^2.5.0, core-js@^2.6.9:
   resolved "https://registry.yarnpkg.com/core-js/-/core-js-2.6.9.tgz#6b4b214620c834152e179323727fc19741b084f2"
   integrity sha512-HOpZf6eXmnl7la+cUdMnLvUxKNqLUzJvgIziQ0DiF3JwSImNphIqdGqzj6hIKyX04MmV0poclQ7+wjWvxQyR2A==
 
-core-js@^3.0.4, core-js@^3.6.5, core-js@^3.8.2, core-js@^3.8.3, core-js@^3.20.2:
+core-js@^3.0.4, core-js@^3.20.2, core-js@^3.6.5, core-js@^3.8.2, core-js@^3.8.3:
```

Comment:
🥳  ... thx so much, will play with it but most likely start of next week, first question, this is an unwanted change I guess? since it seems to be changing the order of this core-js dependencies?

Reply:
I've seen this a few times, I manually updated this line for the last core-js upgrade due to yarn duplicating it.  It's fine to commit, or revert.  I know there's a pretty wide range of yarn patch versions we're allowing - I'll see if that's related.

Reply:
I don't know where it comes from